lib/py/test/test_socket.py

```diff
@@ -25,6 +25,7 @@ def test_isOpen_checks_for_readability(self):
             acc.start()
 
             sock = TSocket(host="localhost", port=acc.port)
+            self.assertFalse(sock.IsOpen())
```

Comment:
this is a typo. it should be `isOpen` instead of `IsOpen`. it's currently causing CI failures: https://app.travis-ci.com/github/apache/thrift/jobs/572946806 & https://app.travis-ci.com/github/apache/thrift/jobs/572946807